integration/common/openlineage/common/provider/dbt.py

```diff
@@ -253,6 +254,17 @@ def node_to_dataset(self, node: Dict, has_facets: bool = False) -> Dataset:
             facets=has_facets
         )
 
+    def extract_fields(self, columns: List[Dict]) -> List[SchemaField]:
+        fields = []
+        for field in columns:
+            type = None
+            if 'data_type' in field and field['data_type'] is None:
+                type = field['data_type']
```

Comment:
do you mean field['data_type'] is **not** None ?
wouldn't type be always None?

Reply:
Yes. Fixed.